README.md

```diff
@@ -260,3 +260,14 @@ will be anonymized for non production environment.
 ## Using services
 
 [See the services README](./app/services/README.md)
+
+
+## <a name="releases"></a>Releases
```

Comment:
What's the `<a>` for?

Reply:
It creates an anchor. Have a look at the "rich diff" version of this document and inspect this heading, and you'll see it emits (unrelated bits elided):

```
<a id="user-content-releases" class="anchor ..." aria-hidden="true" href="...">...</a>
```

So presumably it'll be formatted correctly by GitHub's markdown renderer. Taking an example from the [rails README](https://github.com/rails/rails#model-layer):

```
<a id="user-content-model-layer" class="anchor" aria-hidden="true" href="#model-layer">...</a>